libs/common/src/schemas/tfm/user-upsert-request.schema.ts

```diff
@@ -0,0 +1,13 @@
+import z from 'zod';
+import { TfmTeamSchema } from './tfm-team.schema';
+
+export const USER_UPSERT_REQUEST_SCHEMA = z.object({
```

Comment:
I worry the "upsert request" style naming could be confusing/misleading, what do you think about something like `SSO_USER_REQUEST_SCHEMA` or maybe `ENTRA_...`? Prefer SSO though as it's clearer, not everyone knows what Entra is

Reply:
I am going to try and extract it out to `create_user_request` and alias it from there -- should be a bit clearer.

This request here has nothing to do with entra however -- it's mapped to our database model